vignettes/texreg.Rnw

```diff
@@ -1,1343 +1,1344 @@
-   Arguments		& Description \\
-   \midrule
-   \code{coef.names}	& The names of the independent variables or coefficients. \\
-   \code{coef}		& The actual coefficients. These values must be in the same order as the \code{coef.names}. \\
-   \code{se}		& (\emph{optional}) The standard errors, which will later be put in parentheses. These values must be in the same order as the \code{coef.names}. \\
-   \code{pvalues}	& (\emph{optional}) The $p$~values. They are used to add significance stars. These values must be in the same order as the \code{coef.names}. \\
-   \code{ci.low}	& (\emph{optional}) Lower bounds of the confidence intervals.  An alternative to the \code{se} slot. \\
-   \code{ci.up}		& (\emph{optional}) Upper bounds of the confidence intervals.  An alternative to the \code{se} slot. \\
-   \code{gof.names}	& The names of some GOF statistics to be added to the table. For example, the \code{extract()} method for `\code{lm}' objects extracts R$^2$, Adj.\ R$^2$ and Num.\ obs. \\
-   \code{gof}		& A vector of GOF statistics to be added to the table. These values must be in the same order as the \code{gof.names}. \\
-   \code{gof.decimal}	& (\emph{optional}) A vector of logical (Boolean) values indicating for every GOF value whether the value should have decimal places in the output table. This is useful to avoid decimal places for the number of observations and similar count variables. \\
-   \bottomrule
-  \end{tabular}
- \caption{Arguments of the \code{createTexreg()} function.}
- \label{tab:vectors}
-\end{table}
-
-\subsection{Simple extensions}
-A custom extract function can be easily implemented.  For any model
-type, there exists a function which extracts the relevant information
-from a model.  For example, the \code{extract()} method for `\code{lm}'
-objects provides coefficients and GOF statistics for
-`\code{lm}' objects, the \code{extract()} method for `\code{ergm}' objects
-provides this information for `\code{ergm}' objects, etc.
-
-To get an overview of the model type one is interested in, it is
-recommended to fit a model and examine the resulting object using the
-\code{str(model)} command, the \code{summary(model)} command, the
-\code{summary(model)\$coef} command, and related approaches.
-
-Any new extract function should retrieve the data shown in Table~\ref{tab:vectors} from a statistical model.
-Note that \code{pvalues} and \code{gof.decimal} are optional and can
-be omitted.  Either the \code{se} slot or the \code{ci.low} and
-\code{ci.up} slots must contain values.
-
-Once these data have been located and extracted, a `\code{texreg}'
-object can be created and returned to the \code{texreg()} function.
-The following code provides a simple example for `\code{lm}' objects:
-
-<<results=hide, eval=FALSE>>=
-extract.lm <- function(model) {
-  s <- summary(model)
-  names <- rownames(s$coef)
-  co <- s$coef[, 1]
-  se <- s$coef[, 2]
-  pval <- s$coef[, 4]
-
-  rs <- s$r.squared
-  adj <- s$adj.r.squared
-  n <-
```

Comment:
Aren't they called `Generalized non-linear least squares` models? That's at least compatible with the GNLS abbreviation.

Reply:
yep, I corrected it. Sorry for the inversion!